### PR TITLE
Created about section

### PR DESCRIPTION
**Pull request by NgachaIan**
Finish about section and entire web app

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                <h1 class="header-2">About me</h1>
+                <img class="mask" src="img/Mask Group.png">
+
+                <p class="paragraph"> I can help you build a product , feature or website Look through some of my work
+                    and
+                    experience! If you like
+                    what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
+                <img class="frame" src="img/Frame.png">
```
Comment (Qoosim) [inline index.html:136]: - [ ] If the image is not part of the page content, it should be added via background-image css property. That is what happens here too. Decorations are always backgrounded images, not img tags. Kindly add this background image with css background-image property.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                        <li><a href=""><img src="img/js.png" alt="ruby"></a></li>
+                        <li><a href=""><img src="img/rub.png" alt="css"></a></li>
+                        <li><a href=""><img src="img/ht.png" alt="javascript"></a></li>
+                        <li><a href=""><img src="img/cs.png" alt="html"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item-1">
+                <div>
+                    <img class="icon-language" src="img/icon-frameworks.png">
```
Comment (Qoosim) [inline index.html:153]: - [ ] If the image is not part of the page content, it should be added via background-image css property. That is what happens here too. Decorations are always backgrounded images, not img tags. Kindly add this background image with css background-image property.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                <div class="grid-item-bottom">
+                    <h4 class="header-1">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="image-container">
+                        <li><a href=""><img src="img/ruby.png" alt="ruby"></a></li>
+                        <li><a href=""><img src="img/css.png" alt="css"></a></li>
+                        <li><a href=""><img src="img/javascript.png" alt="javascript"></a></li>
+                        <li><a href=""><img src="img/html.png" alt="html"></a></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
```
Comment (Qoosim) [inline index.html:59]: - [ ] Kindly remove the dark border on the `See project` button so as to match the Figma design.

### Your Design
your-design

### Figma Design
temp-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @NgachaIan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @NgachaIan, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>


## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**